centaur/src/test/scala/centaur/s3filechecking/TestAwsFileChecker.scala

```diff
@@ -0,0 +1,50 @@
+package centaur.s3filechecking
+
+import java.util
+
+import centaur.test.AwsS3Ops
+import com.amazonaws.services.s3.AmazonS3
+import com.amazonaws.services.s3.model.{ObjectListing, S3ObjectSummary}
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.FlatSpec
+import org.specs2.matcher.ShouldMatchers
+
+class TestAwsFileChecker extends FlatSpec with ShouldMatchers with MockFactory {
```

Comment:
Please, merge these two specifications (`NegTestAwsFileChecker` and `TestAwsFileChecker`) into a single one with all the negative and positive cases.

In addition, all the specifications usually are named by the following: the name of the entity that is tested plus **Spec** suffix (for instance: `ListSpec`, `UserControllerSpec`, etc).